lib/linguist/languages.yml

```diff
@@ -730,6 +730,18 @@ Closure Templates:
   - ".soy"
   tm_scope: text.html.soy
   language_id: 357046146
+CoNLL-U:
+  type: data
+  extensions:
+  - ".conllu"
+  - ".conll"
+  - ".conllx"
```

Comment:
I'm seeing this only in my second review, but it looks like all `.conllx` files are [from a single user](https://github.com/search?p=1&q=extension%3Aconllx+NOT+djfhdjfgnd&type=Code&utf8=%E2%9C%93). I'm afraid you'll have to remove it as it doesn't classify as widespread (in use in hundreds of repository).

Reply:
fixed in 2aaee98! 

indeed, I hadn't noticed that! the CoNLL-X format is actually a very similar format standard that has been superseded by the CoNLL-U format, so it seems that no one is using it anymore !